torch/_dynamo/output_graph.py

```diff
@@ -1003,7 +1003,10 @@ def compile_subgraph(
                 gb_type="Attempt to compile graph in a try block",
                 context="",
                 explanation="Dynamo cannot compile traced graphs while in a try block.",
-                hints=[],
+                hints=[
+                    *graph_break_hints.FUNDAMENTAL,
```

Comment:
supportable?

Reply:
Supportable gives the message that "we could probably support this, but we haven't gotten around to doing it yet," but I'm not sure if supporting graph breaks in try blocks is actually possible (it might be in 3.11+?). Don't want to make such a promise in this hint - so we can leave it empty then.

Reply:
Yeah fundamental one also seems wrong.  It is not impossible to support.